appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -31,7 +31,10 @@
 from oauth2client.client import GoogleCredentials
 
 
-_FIREBASE_CONFIG = '_firebase_config.html'
+import rest_api
```

Comment:
Feel free to just kill this line. Was only using for testing. Don't actually want to break into a separate module. I created the file solely to allow the code for the REST API to be surfaced in the article.